src/Hydrators/IngredientEntityHydrator.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+
+namespace Mamoi\Entities;
```

Comment:
The namespace should be 'Mamoi\Hydrators' - my mistake in the example!

Reply:
ooh sorry yes! my lousy copy and pasting there oops!!